base issue not updated & minor theme issue

rosscdh opened the pull request:
fixed themes issue and base issue not updated on new sub issue

adamcooke commented:
I'm afraid I cannot merge this. 

* The update time is fixed already (see issue #115).
* This also forces some unchangeable SMTP configuration which is not desirable.
* What is the issue you're trying to solve with adding the default theme to the precompile lines? I've never had an issue with the theme being compiled.

rosscdh commented:
1. great will update locally
2. yes agreed (as i noted in my comments)
3. i simply coped default theme and renamed made my changes... but woudl not complile the "new" assets. I added these two lines as prescribed by the rake task and voila working again
